src/org/zaproxy/zap/extension/jython/JythonOptionPanel.java

```diff
@@ -0,0 +1,120 @@
+
+	private JTextField modulesPathTextField;
+	private JButton pathChooseButton;
+
+	public JythonOptionPanel() {
+		super();
+		this.initComponents();
+	}
+
+	@Override
+	public String getHelpIndex() {
+		return null;
+	}
+
+	@Override
+	public void initParam(Object object) {
+		JythonOptionParam jythonOptionParam = ((OptionsParam) object).getParamSet(JythonOptionParam.class);
+		this.modulesPathTextField.setText(Strings.nullToEmpty(jythonOptionParam.getModulePath()));
+	}
+
+	@Override
+	public void saveParam(Object object) throws Exception {
+		JythonOptionParam jythonOptionParam = ((OptionsParam) object).getParamSet(JythonOptionParam.class);
+		jythonOptionParam.setModulePath(Strings.nullToEmpty(this.modulesPathTextField.getText()));
+	}
+
+	@Override
+	public void validateParam(Object object) throws Exception {
+		String modulesPathString = this.modulesPathTextField.getText();
+		if (false == Strings.isNullOrEmpty(modulesPathString)) {
+			File modulesPath = new File(modulesPathString);
+			if (false == modulesPath.exists()) {
+				throw new NoSuchFileException(
+						Constant.messages.getString("jython.options.error.modulepath.notexist", modulesPath));
+			}
+			if (false == modulesPath.isDirectory()) {
+				throw new NotDirectoryException(
+						Constant.messages.getString("jython.options.error.modulepath.notdirectory", modulesPath));
+			}
+		}
+	}
+
+	private void initComponents() {
+		super.setName(Constant.messages.getString("jython.options.title"));
+
+		JLabel modulesPathLabel = new JLabel(Constant.messages.getString("jython.options.label.modulepath"));
+		this.modulesPathTextField = new JTextField();
+		this.pathChooseButton = new JButton(Constant.messages.getString("jython.options.label.choose"));
+		this.pathChooseButton.addActionListener(this);
+		JLabel cautionLabel = new JLabel(Constant.messages.getString("jython.options.label.caution"));
+
+		GroupLayout layout = new GroupLayout(this);
+		super.setLayout(layout);
+		layout.setAutoCreateGaps(true);
+		layout.setAutoCreateContainerGaps(true);
+		layout.setHorizontalGroup(layout.createParallelGroup(GroupLayout.Alignment.LEADING)
+				.addGroup(layout.createSequentialGroup().addComponent(modulesPathLabel)
+						.addComponent(this.modulesPathTextField).addComponent(this.pathChooseButton))
+				.addGroup(layout.createSequentialGroup().addComponent(cautionLabel)));
+		layout.setVerticalGroup(layout.createSequentialGroup()
+				.addGroup(layout.createParallelGroup(GroupLayout.Alignment.BASELINE).addComponent(modulesPathLabel)
+						.addComponent(this.modulesPathTextField).addComponent(this.pathChooseButton))
+				.addGroup(layout.createParallelGroup().addComponent(cautionLabel)));
+	}
+
+	@Override
+	public void actionPerformed(ActionEvent actionEvent) {
+		if (actionEvent.getSource().equals(this.pathChooseButton)) {
+			JFileChooser chooser = new JFileChooser();
+			chooser.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
+			chooser.setAcceptAllFileFilterUsed(false);
```

Comment:
The current directory(ZAP installed directory?) would be not the Python module path, and theirs distance would be far.
Also the module path varies according to the environment.
So, I think there is no need to specify the current directory.